core/tests/test_api.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+from conftest import CreateCommunitiesFixture, create_posts
+from core.models import Community, Post
+from reddit import settings
+
+pytestmark = pytest.mark.django_db
+
+
+PAGE_SIZE = 10
+PREFIX = "http://testserver"
+
+
+def get_abs_url(url: str, page: int) -> str:
+    return f"{PREFIX}{url}?page={page}"
+
+
+def test_pagination_size() -> None:
+    assert settings.REST_FRAMEWORK["PAGE_SIZE"] == PAGE_SIZE
```

Comment:
Please, do not test settings. If we change page size this will break which is bad. No resiliency in test. 
For testing pagination you may use test specific setting.

Reply:
As suggested below by AI.